Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -295,3 +400,97 @@ extension SecureEnclaveValet {
     }
 
 }
+
+// MARK: - Private Extensions
+
+extension SecureEnclaveAccessControl {
+
+    /// The `LAPolicy` that the `SecureEnclaveAccessControl` corresponds to.
+    #if !os(watchOS)
+    @available(tvOS 10.0, *)
+    fileprivate var policy: LAPolicy {
+        switch self {
+        case .userPresence, .devicePasscode:
+            return .deviceOwnerAuthentication
```

Comment:
so I'm not sure what to do about this, but [`.deviceOwnerAuthentication`](https://developer.apple.com/documentation/localauthentication/lapolicy/deviceownerauthentication) doesn't exactly match either `.userPresence` or `.devicePasscode`.

`.deviceOwnerAuthentication` gives the option of using device passcode, biometrics, or watch. Meanwhile, `.devicePasscode` doesn't allow biometrics, and `.userPresence` doesn't allow for watch (at least, I'm pretty sure about that last bit).

In terms of a path forward, I think we'll need to document here and in the new API how this API can enable more authentication options than the `SecureEnclaveAccessControl` set might allow. It's not great, but... in a world where Apple hasn't given us more options I think that's a reasonable path forward.

Reply:
Hmm good point. Maybe this function shouldn't be available with `SecureEnclaveAccessControl. devicePasscode`, but I think we can use this for `userPresence` - the documentation for `userPresence` says "On macOS 10.15 and later, this element may also be accessed via a prompt on a paired watch"

Reply:
I'm going to throw an error for `devicePasscode`, let me know what you think

Reply:
Throwing an error on `devicePasscode` makes sense. We gotta document that with a `/// - Warning:` comment maybe but I like that approach.